web/src/components/netflow-table.tsx

```diff
@@ -37,71 +36,49 @@ const NetflowTable: React.FC<{
   // sort direction of the currently active column
   const [activeSortDirection, setActiveSortDirection] = React.useState<string>('asc');
 
-  //Sort handler
+  // sort function
+  const getSortedFlows = () => {
+    if (activeSortIndex < 0 || activeSortIndex >= columns.length) {
+      return flows;
+    } else {
+      return flows.sort((a, b): number => {
```

Comment:
Just a remark, I think it can be optimized a bit by factoring out all the `switch` block out of the sort function, so that this logic isn't repeated for each comparison that `sort` does. Maybe we could even define sort functions per field in `column.ts`, and here we would just call something like:

```typescript
    return flows.sort(columns[activeSortIndex].sort)
```

You see what I mean? wdyt?

Reply:
I like the idea ! I'll take a look at it.

Thanks